api/kv_v2.go

```diff
@@ -30,6 +30,26 @@ type KVMetadata struct {
 	Raw      *Secret
 }
 
+// KVMetadataInput is the subset of metadata that can be manually modified for
+// a KV v2 secret. This can also be passed to the PutMetadata method to create
+// a secret without any data yet.
+//
+// The struct's fields are all pointers. When used with the PatchMetadata
+// method, a pointer to a field's zero value (e.g. false for *bool) implies
+// that field should be reset to its zero value after update, whereas a field
+// set to a null pointer (e.g. null for *bool) implies the field should remain
+// unchanged.
+//
+// Since maps are already pointers, use an empty map to remove all
+// custom metadata. To explicitly set the zero value for time.Duration,
+// you can use a pointer to time.ParseDuration("0s").
```

Comment:
I believe the default value of time.Duration is a zero value, so no need to use Parse.